### PR TITLE
cleanup json type

### PR DESCRIPTION
**Pull request by zhexuany**

### CODE REVIEW

File: tikv-client/src/main/java/com/pingcap/tikv/types/JsonType.java [hunk 1]
```diff
@@ -137,37 +138,26 @@ private JsonElement parseValue(byte type, DataInput di) {
   // * notice use this as a unsigned long
   private long parseUint64(DataInput cdi) {
     byte[] readBuffer = new byte[8];
-    readFully(cdi, readBuffer, 0, 8);
+    readFully(cdi, readBuffer);
+
+    return getaLong(readBuffer);
+  }
 
+  private long getaLong(byte[] readBuffer) {
```
Comment (ilovesoup) [inline tikv-client/src/main/java/com/pingcap/tikv/types/JsonType.java]: geta?

File: tikv-client/src/main/java/com/pingcap/tikv/types/JsonType.java [hunk 2]
```diff
@@ -137,37 +138,26 @@ private JsonElement parseValue(byte type, DataInput di) {
   // * notice use this as a unsigned long
   private long parseUint64(DataInput cdi) {
```
Comment (ilovesoup) [inline tikv-client/src/main/java/com/pingcap/tikv/types/JsonType.java]: What is the difference btw this one and the one inside CDI?

**Comment by ilovesoup** [comment]
Also this file has no copyright. Please port this change into client project or log an issue.

File: tikv-client/src/main/java/com/pingcap/tikv/types/JsonType.java (hunk 2 above)
Comment (zhexuany) [reply tikv-client/src/main/java/com/pingcap/tikv/types/JsonType.java]: I am not sure. I prefer not to change the logic if we only do a cleanup.

**Comment by zhexuany** [comment]
closed since @birdstorm takes over.

**Pull request closed without merging**